packages/graphql-playground-html/src/render-playground-page.ts

```diff
@@ -89,7 +89,7 @@ const loading = getLoadingMarkup()
 const CONFIG_ID = 'playground-config';
 
 const getCdnMarkup = ({ version, cdnUrl = '//cdn.jsdelivr.net/npm', faviconUrl }) => {
-  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}/${version ? `@${version}/` : ''}${suffix}` || '')
+  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}${version ? `@${version}/` : ''}${suffix}` || '')
```

Comment:
Hi @tgroutars ,
I was about to raise a PR and then found your PR, thanks for raising the PR.

I'd like to provide a typo fix here in the sense that when a `version` is not there, we still need `/`.

```suggestion
  const buildCDNUrl = (packageName: string, suffix: string) => filter(`${cdnUrl}/${packageName}${version ? `@${version}/` : '/'}${suffix}` || '')
```